gitlab/v4/cli.py

```diff
@@ -88,46 +105,75 @@ def do_project_export_download(self):
         except Exception as e:
             cli.die("Impossible to download the export", e)
 
-    def do_create(self):
+    def do_create(self) -> gitlab.base.RESTObject:
+        if TYPE_CHECKING:
+            assert isinstance(self.mgr, gitlab.mixins.CreateMixin)
         try:
-            return self.mgr.create(self.args)
+            result = self.mgr.create(self.args)
```

Comment:
Yes. If we don't do that then `mypy` doesn't understand that the only return value is result of the `return` statement.

Basically it will flag it as an error on the return value.